README.md

```diff
@@ -1,6 +1,40 @@
 # NotDog
-> "What would you say if I told you there is a app on the market that tell you if you have a dog or not a dog. It is very good and I do not want to work on it any more. You can hire someone else." - Jian Yang, 2017
 
-[NotDog](https://not-dog-io.firebaseapp.com) is the Shazam of animals. It is based on the [Not HotDog app](https://www.seefoodtechnologies.com/nothotdog/) developed by See Food Technologies.
+[NotDog](https://not-dog-io.firebaseapp.com) is the Shazam of animals. It tells you whether you have a dog or not.
+It is based on the [Not HotDog app](https://www.seefoodtechnologies.com/nothotdog/) developed by See Food Technologies.
 
-The application uses [ML Kit for Firebase](https://firebase.google.com/products/ml-kit) to tell you if you have a dog or not.
+
+<p align="center">
+  <img src="screenshots/negative_notdog.jpg" height="480" width="270" alt="Not Dog"/>
+  <img src="screenshots/positive_notdog.jpg" height="480" width="270" alt="Not Dog"/>
+</p>
+
+# Getting Started
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. 
+
+## Prerequisites
+Before building the project, you will need:
+- A [Firebase Project](https://firebase.google.com/)
+- A recent version of Android Studio (v3.0+)
+- A Minimum SDK level of 14 or higher
+
+## Installing 
+To get the app up and running, you have to:
+1. [Create a new Firebase Android App](https://firebase.google.com/docs/android/setup)
+2. Add the SH-1 Key and the app ID
+3. Download the ```google-services.json```
+4. Build the project
```

Comment:
Seems like we're skipping a few steps here. The google-services.json file should be copied to the app folder.